source/megengine.js

```diff
@@ -698,4 +1032,4 @@ megengine.Error = class extends Error {
 
 if (typeof module !== 'undefined' && typeof module.exports === 'object') {
     module.exports.ModelFactory = megengine.ModelFactory;
-}
```

Comment:
Add test files to `./test/models.json` and run `node ./test/models.json megengine` successfully.

Reply:
done

Reply:
File extension of `target` should be the same as the extracted file?

```
  {
    "type":     "megengine",
    "target":   "shufflenetv2.tm",
    "source":   "https://github.com/lutzroeder/netron/files/9441551/shufflenetv2.zip[shufflenetv2.mge]",
```

Reply:
done

Reply:
Also sort by `type` > `target`.

Reply:
done

Reply:
done